circle.yml

```diff
@@ -1,14 +1,29 @@
 machine:
+  services:
+    - docker
   node:
     version: 7.7.1
+
+dependencies:
+  pre:
+    # Required for integration tests of ilp-plugin-xrp-paychan
+    - > 
+      mkdir test/integration && 
+      wget -P test/integration https://raw.githubusercontent.com/ripple/ilp-plugin-xrp-paychan/da-integration-tset/test/integration/Dockerfile && 
+      wget -P test/integration https://raw.githubusercontent.com/ripple/ilp-plugin-xrp-paychan/da-integration-tset/test/integration/rippled.cfg && 
```

Comment:
This seems a little janky, but I don't know if there's a better way to do it. @justmoon want to weigh in on this?

Reply:
Documented this in #62